rel/ensmallen-release.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Hm, should we use ``#!/usr/bin/env bash`` here to be more portable?

Reply:
Good point, thanks!  Fixed in 083a9a6.